### ISSUE TITLE
Outdated / missing build guide

### ISSUE THREAD

**Issue by TAOcroatia** [vasishath/multirom#28]
Sorry if this is not the right place, but i am not really aware how do i build this. README isn't very useful, as i see it mentions android 4.4.

**Comment by vasishath** [comment]
The README is out of date since long.. ( I actually never even noticed that it exists tbh). Anyways, this is a native cpp project to be built with android AOSP source code and hence does not build with Android studio or need a build.gradle file.

**Comment by TAOcroatia** [comment]
So I need to download 150GB of android source to build?

**Comment by vasishath** [comment]
Not exactly.. u can go for minimal-manifest-twrp which syncs only limited repositories needed to build twrp recovery

**Comment by TAOcroatia** [comment]
Thanks for the tip. This is the output i get when running `make -j4 multirom trampoline` https://pastebin.com/dB4tPNxP

I'm thinking of making a pull request with rewritten README and some instructions for building (once i figure it out for myself of course), so more people can contribute to the project
